RocketChat/Rocket.Chat.ReactNative
# regression: open self profile from messages

## Proposed changes


Regression from  https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5488

We need to pass the itsMe to return the roomUserId's value correctly

- Also fixed the name of the inAppFeedback reducer

## Issue(s)	


## How to test or reproduce


- Send a message and click on your own avatar, username or mention.
- You should be moved properly to the User Info

## Screenshots

#### Before

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/5728f991-3d2d-4e8a-9b1f-1d244f03498f


#### After

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/b3cc97e1-c8ee-4929-86e7-31bb48d7d582

## Types of changes




## Checklist



## Further comments

[CORE-54]

[CORE-54]: https://rocketchat.atlassian.net/browse/CORE-54?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
